plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -145,7 +147,8 @@ class PodDebugTabView extends React.Component {
     let mainContent = null;
     let offerCount = null;
 
-    if (queue == null || queue.declinedOffers.summary == null) {
+    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(queue)
+      || queue.declinedOffers.summary == null) {
```

Comment:
Couldn't we default assign these values and let go of the `if`-`else` here?
Not part of this PR, but would be nice with a little cleanup here.